src/types.rs

```diff
@@ -100,15 +99,25 @@ impl<T> Point<T>
         self.0.y = y;
         self
     }
+}
 
+impl<T> Point<T>
+    where T: Angle
+{
     /// Returns the longitude/horizontal component of the point.
     ///
     /// ```
+    /// # extern crate geo;
+    /// # extern crate cgmath;
+    /// #
     /// use geo::Point;
+    /// use cgmath::Deg;
     ///
-    /// let p = Point::new(1.234, 2.345);
+    /// # fn main() {
+    /// let p = Point::new(Deg(1.234), Deg(2.345));
     ///
-    /// assert_eq!(p.lng(), 1.234);
+    /// assert_eq!(p.lng(), Deg(1.234));
+    /// # }
     /// ```
     pub fn lng(&self) -> T {
```

Comment:
Why is `lng` only implemented for implementors of `Angle`? What about for other types that implement `Float`?